doc/source/tutorials/wrf_workshop_2019.rst

```diff
@@ -81,8 +81,7 @@ For Linux:
 Step 2: Download Miniconda
 ----------------------------
 
-For this tutorial, you will need to download and install Miniconda.  We are 
-going to use Python 3.7, but it will also work with Python 2.7. 
+For this tutorial, you will need to download and install Miniconda.
```

Comment:
```suggestion
For this tutorial, you will need to download and install Miniconda.  We are 
going to use Python 3.7, but it will also work with Python 2.7.
```